demos/fab.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
```

Comment:
Done, added an "edit" icon that also uses SVG to test that out. Had to add `fill: currentColor` to the block to handle giving them the proper color. ;)